pandas/formats/format.py

```diff
@@ -197,7 +197,8 @@ def _get_footer(self):
                                        escape_chars=('\t', '\r', '\n'))
             footer += ("Name: %s" % series_name) if name is not None else ""
 
-        if self.length:
+        if (self.length is True or
```

Comment:
Yeah, I actually misuse the `length` keyword here a bit, which is normally a True/False keyword, but by internally also passing 'truncate' to it (otherwise I had to create a new keyword show_dimensions, which would render `length` useless)